code/modules/antagonists/paradox_clone/datum_paradox_clone.dm

```diff
@@ -0,0 +1,177 @@
+GLOBAL_LIST_INIT(possible_paradox_clone_ids, list("Alpha", "Bravo", "Charlie",
+	"Delta", "Echo", "Foxtrot", "Golf", "Hotel", "India", "Juliet",
+	"Kilo", "Lima", "Mike", "November", "Oscar", "Papa", "Quebec",
+	"Romeo", "Sierra", "Tango", "Uniform", "Victor", "Whiskey", "X-ray",
+	"Yankee", "Zulu")) // just a copy of GLOB.photenic_alphabet but for paradox clones ids which removes
+
+GLOBAL_LIST_INIT(paradox_clones, list())
+
+/datum/antagonist/paradox_clone
+	name = "Paradox Clone"
+	roundend_category = "paradox clones"
+	job_rank = ROLE_PARADOX_CLONE
+	special_role = SPECIAL_ROLE_PARADOX_CLONE
+	antag_hud_name = "hudparadoxclone"
+	antag_hud_type = ANTAG_HUD_PARADOX_CLONE // they see each other hud
+	clown_gain_text = "<span class='notice'>You are from alternative universe, where clowns are actually quite dextrous. This allows you to wield weapons without harming yourself.</span>"
+	clown_removal_text = "<span class='danger'>Your connection with your native universe fades, and with it goes your dexterity.</span>"
+	wiki_page_name = "Paradox_Clone"
+	var/paradox_id = "Paradox"
+	var/real_id
+	var/static/list/paradox_powers = list(
+		/datum/spell/paradox_spell/click_target/gaze,
+		/datum/spell/touch/paradox_spell/space_distortion,
+		/datum/spell/paradox_spell/self/digital_supersede,
+		/datum/spell/paradox_spell/click_target/mind_interference,
+		/datum/spell/paradox_spell/self/illusion,
+		/datum/spell/paradox_spell/click_target/energy_exchange,
+		/datum/spell/touch/paradox_spell/microcircuit_disorder,
+		/datum/spell/paradox_spell/self/intangibility,
+		/datum/spell/touch/paradox_spell/suppression,
+		/datum/spell/paradox_spell/aoe/hud_malfunction
+	)
+	// "replace" is issued if there is a objective to kill n replace and "United Bonds" when need to protect original.
+	var/mob/living/carbon/human/original
+	var/list/current_powers = list()
+	give_objectives = FALSE
+	silent = TRUE
+
+	blurb_text_color = COLOR_GRAY
+	blurb_text_outline_width = 2
+
+	blurb_r = 46
+	blurb_g = 46
+	blurb_b = 46
+	blurb_a = 0.40
```

Comment:
Formatting nitpick, please move all of these above the new var declarations like `var/paradox_id` etc.